docs/installation/Docker.md

```diff
@@ -0,0 +1,20 @@
+# Docker based installation
+
+Nussknacker is available at [Docker Hub](https://hub.docker.com/r/touk/nussknacker/).
+You can check an example installation
+with docker-compose at [installation example repository](https://github.com/TouK/nussknacker-installation-example/). 
+
+Please note, in the example above the Lite engine is run inside the Designer. It's not a production setup and for production 
+you will need configured Kubernetes cluster to actually run scenarios on the Lite engine - we recommend using Helm installation for that mode.
```

Comment:
1. `for that mode.` - I'd prefer not to use `mode` in this context. Also saying that its run `inside the Designer` may be a not needed. How about something like:

`
Please note, the example above provides only a non-production ready way to run scenarios on Lite engine. For Lite engine production setup, you will need a configured Kubernetes cluster.  We recommend using Helm installation.
`

2. WDYT about instead of `please note` make it visual through a `:::note` block? 
3. when referring to `helm installation` adding a link to HelmChart.md would be nice


This also applies to similar paragraph in Binaries.md (https://github.com/TouK/nussknacker/pull/6380/files#diff-ed2bd6b2676087ad5062b8cb0095b79810a54959cf53ab32b8f182bb24090286R6)